conf/modules/extra_dl.xml

```diff
@@ -10,6 +10,7 @@
     <file name="extra_pprz_dl.h"/>
   </header>
   <init fun="extra_pprz_dl_init()"/>
+  <periodic fun="extra_pprz_dl_periodic()" freq="PERIODIC_FREQUENCY" autorun="TRUE"/>
```

Comment:
I was using ```<configure name="TELEMETRY_FREQUENCY" value="PERIODIC_FREQUENCY""/>``` for all targets, because this is not only for NPS, has to work like that for AP too...